src/overlays/actors/ovl_En_Ru2/z_en_ru2.c

```diff
@@ -499,56 +633,74 @@ void func_80AF321C(EnRu2* this, PlayState* play) {
     CLOSE_DISPS(play->state.gfxCtx, "../z_en_ru2_inKenjyanomaDemo02.c", 291);
 }
 
-void func_80AF3394(EnRu2* this, PlayState* play) {
-    func_80AF28E8(this, &gAdultRutoIdleHandsOnHipsAnim, 0, 0.0f, 0);
-    this->action = 10;
-    this->drawConfig = 0;
+/**
+ * Sets up Ruto's hands-on-hips pose during the credits sequence.
+ */
+void EnRu2_InitCredits(EnRu2* this, PlayState* play) {
+    EnRu2_AnimationChange(this, &gAdultRutoIdleHandsOnHipsAnim, 0, 0.0f, 0);
+    this->action = ENRU2_CREDITS_INVISIBLE;
+    this->drawConfig = ENRU2_DRAW_NOTHING;
     this->actor.shape.shadowAlpha = 0;
 }
 
-void func_80AF33E0(EnRu2* this) {
-    f32* unk_2B0 = &this->unk_2B0;
+/**
+ * Fades in Ruto's actor during the credits sequence.
+ */
+void EnRu2_FadeInCredits(EnRu2* this) {
+    f32* fadeTimer = &this->fadeTimer;
     f32 temp_f0;
     s32 temp_f18;
 
-    *unk_2B0 += 1.0f;
+    *fadeTimer += 1.0f;
 
     temp_f0 = kREG(17) + 10.0f;
-    if (temp_f0 <= *unk_2B0) {
+    if (temp_f0 <= *fadeTimer) {
         this->alpha = 255;
         this->actor.shape.shadowAlpha = 0xFF;
     } else {
-        temp_f18 = (*unk_2B0 / temp_f0) * 255.0f;
+        temp_f18 = (*fadeTimer / temp_f0) * 255.0f;
         this->alpha = temp_f18;
         this->actor.shape.shadowAlpha = temp_f18;
     }
 }
 
-void func_80AF346C(EnRu2* this, PlayState* play) {
-    func_80AF2868(this, play, 3);
-    this->action = 11;
-    this->drawConfig = 2;
+void EnRu2_InitCreditsPosition(EnRu2* this, PlayState* play) {
+    EnRu2_InitPositionFromCue(this, play, 3);
+    this->action = ENRU2_CREDITS_FADE_IN;
+    this->drawConfig = ENRU2_DRAW_XLU;
 }
 
-void func_80AF34A4(EnRu2* this) {
-    if (this->unk_2B0 >= kREG(17) + 10.0f) {
-        this->action = 12;
-        this->drawConfig = 1;
+/**
+ * Checks if Ruto's actor is fully faded in during the credits sequence, and if so, proceeds to action 12.
+ */
+void EnRu2_CheckVisibleInCredits(EnRu2* this) {
+    if (this->fadeTimer >= kREG(17) + 10.0f) {
+        this->action = ENRU2_CREDITS_VISIBLE;
+        this->drawConfig = ENRU2_DRAW_OPA;
     }
 }
 
-void func_80AF34F0(EnRu2* this) {
-    func_80AF28E8(this, &gAdultRutoHeadTurnDownLeftAnim, 2, 0.0f, 0);
-    this->action = 13;
+/**
+ * Starts Ruto's animation to look down towards Nabooru during the credits sequence.
+ */
+void EnRu2_TurnHeadDownLeftAnimation(EnRu2* this) {
```

Comment:
These kinds of functions are usually named "SetupX" where X is the new action being changed to.
I think it would be nice if it conformed to the style from other actors, but do understand that the cutscene actors are written a bit differently than most actors